README.md

```diff
@@ -49,45 +49,28 @@ Now the app should be served at `http://127.0.0.1:3000`
 
 * All server functions (`#[server]`) **must** be explicitly registered (see usage sample below). In native code, Leptos uses a clever macro to register them automatically; unfortunately, that doesn't work in WASI.
 
-* When using a context value in a component in a `feature = "ssr"` block, you **must** call `use_context` **not** `expect_context`. `expect_context` will panic during routing.  E.g.
```

Comment:
Did you remove this section for a reason?

Reply:
I was seeing this work with expect_context in testing -- for example [in the NotFound component](https://github.com/fermyon/leptos-spin/pull/48/files#diff-782be6a82ee08bd1b4914a1c0ca4dba43937f23e80b5d065f06287678ef51ee1R81)

Perhaps this is still unsupported though and I was misreading that behavior?

Reply:
We definitely still do route generation that could not have it, but depending on what you're getting it may or may not work. Request or something from Request is the most likely to hit this issue. Assuming it still is, this could be reworded and clarified

Reply:
Ah good to know! Updated to re-add this section with a bit of additional context